packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -102,39 +97,12 @@ export function getReqSerializers(): ReqSerializers<Api, ReqTypes> {
 }
 
 export function getReturnTypes(): ReturnTypes<Api> {
-  const lightclientSnapshotWithProofType = new ContainerType(
```

Comment:
same here. Any reason to have these types redefined instead of using the ssz types?

Reply:
Same. Best to just use the ssz types and keep that as the single source of truth